cmd/snap-exec/main_test.go

```diff
@@ -98,15 +103,31 @@ func (s *snapExecSuite) TestFindCommand(c *C) {
 	info, err := snap.InfoFromSnapYaml(mockYaml)
 	c.Assert(err, IsNil)
 
+	snapPath := fmt.Sprintf("%s/snapname/unset", dirs.SnapMountDir)
+
 	for _, t := range []struct {
-		cmd      string
-		expected string
+		app          string
+		cmd          string
+		includeChain bool
```

Comment:
Given by how it's used in the test, it should be called `skipChain` or somesuch.

Reply:
Ah, good catch, you can tell I switched that around halfway through!